lib/logger.ts

```diff
@@ -54,6 +54,8 @@ export class Logger {
   static set(config: Config): void {
     if (config.troubleshoot) {
       Logger.logLevel = LogLevel.DEBUG;
+    } else if (config.logLevel) {
+      Logger.logLevel = (<any>LogLevel)[config.logLevel];
```

Comment:
NVM this is an error in `typescript@2.2.1` but not `2.0.10` (which we have pinned)